tests/cache/tests.py

```diff
@@ -1144,6 +1144,28 @@ def test_incr_decr_timeout(self):
         cache.decr(key)
         self.assertEqual(expire, cache._expire_info[_key])
 
+    @override_settings(CACHES=caches_setting_for_tests(
+        BACKEND='django.core.cache.backends.locmem.LocMemCache',
+        OPTIONS={'MAX_ENTRIES': 30},
+    ))
+    def test_lru_eviction_policy(self):
+        "Items are evicted in LRU order."
```

Comment:
We use triple quotes for new docstrings.

Reply:
Fixed.